### PR TITLE
hide debug outputs

### PR DESCRIPTION
**Pull request by shinji-kohara**
To remove debug message it's displayed on debug environment, comment out messages.

This PR won't be merged to master because current main stream version is 3.x but this is a successor of 2.1.x.

### CODE REVIEW

File: libs/Gengo/Client.php [hunk 1]
```diff
@@ -142,8 +142,8 @@ protected function request($url, $method, $format = null, $params = null)
             if ($this->config->get('debug', false))
             {
                 $response = $this->client->request($method);
-                echo $this->client->getUri(true) . "\n";
-                echo $this->client->getLastRequest() . "\n";
+                echo "<!-- " . $this->client->getUri(true) . " -->\n";
+                echo "<!-- " . $this->client->getLastRequest() . " -->\n";
```
Comment (markjezreeldagot) [inline libs/Gengo/Client.php:146]: how about we will delete it?

Comment (matthias-mess) [inline libs/Gengo/Client.php:146]: Maybe we could just remove this messages? Not sure what it is for.

**Comment by shinji-kohara** [comment]
@markjezreeldagot @matthias-mess I think either's fine. I just thought it may convenient for some purpose because it have been remained long time.
well, let's remove these statements if we don't need it.

**Review by matthias-mess — APPROVED** [review]

**Comment by shinji-kohara** [comment]
close this PR because this shouldn't be merged into master.
this should be remained as a branch because v2.1.x isn't main stream anymore.

**Pull request closed without merging**